dao/src/test/java/com/fitgoal/dao/impl/AbstractMongoTestContainersTest.java

```diff
@@ -0,0 +1,46 @@
+package com.fitgoal.dao.impl;
+
+import com.github.dockerjava.api.model.HostConfig;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.ServerAddress;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import org.junit.ClassRule;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+
+import org.testcontainers.containers.GenericContainer;
+
+import java.util.Collections;
+import java.util.function.Consumer;
+
+public abstract class AbstractMongoTestContainersTest {
+    private static final String MONGO_VERSION = "4.2.1";
+    private static final int MONGO_PORT = 27017;
+    private static final int MONGO_HOST_PORT = 27019;
+
+    private static Consumer<CreateContainerCmd> hostConfig = e -> e.withHostConfig(
```

Comment:
it's a bad practice to use such short and meaningless names and lambda arguments like `e`

Reply:
Refactored